frontends/p4/fromv1.0/converters.h

```diff
@@ -759,6 +759,18 @@ class InsertCompilerGeneratedStartState: public Transform {
         return state;
     }
 
+    // Rename any path refering to original start state
+    const IR::Node* postorder(IR::Path* path) override {
```

Comment:
won't this also rename a variable reference if the variable is named "start"?
To be safe this should check the parent context to be a SelectCase, and moreover this to be the child #1.

Reply:
Updated to explicitly check if the path that is being changed is either within:
`ParserState->selectExpression<PathExpression>->path`
Or:
`SelectCase->state<PathExpression>->path`